TradeItIosTicketSDK2/AdService.swift

```diff
@@ -61,7 +61,7 @@ import UIKit
 
```

Comment:
@jsom Question about the comment that was there before on `labelFor` in the above enums about obj-c backwards compatibility.

When Swift enums are exposed to obj-c, only their raw types are visible, right? So if obj-c cannot see this `labelFor` method, what's the comment re:compatibility about?

Reply:
Obj-c does not support Swift non-integer enums AFAIK. If we didn't need to support obj-c then we could make this a `public enum TradeItAdPosition: String` and get rid of the `labelFor` - I haven't looked at this for a while though so maybe something changed.